src/modules/client/components/ClientAlerts.tsx

```diff
@@ -0,0 +1,150 @@
+      body: `${theme.palette.error.main}04`,
+      border: `${theme.palette.error.main}50`,
+    },
+    medium: {
+      header: theme.palette.alerts.mediumPriorityHeaderBackground,
+      body: theme.palette.alerts.mediumPriorityBodyBackground,
+      border: theme.palette.alerts.mediumPriorityBorder,
+    },
+    low: {
+      header: theme.palette.alerts.lowPriorityHeaderBackground,
+      body: theme.palette.alerts.lowPriorityBodyBackground,
+      border: theme.palette.alerts.lowPriorityHeaderBackground,
+    },
+  };
+
+  return (
+    <Alert
+      sx={{
+        p: 0,
+        color: 'black',
+        backgroundColor: priorityColors[priority].body,
+        borderColor: priorityColors[priority].border,
+        '& .MuiAlert-message': {
+          p: 0,
+          width: '100%',
+        },
+      }}
+      icon={false}
+      variant='outlined'
+    >
+      <AlertTitle
+        sx={{
+          p: 2,
+          m: 0,
+          backgroundColor: priorityColors[priority].header,
+          fontWeight: 'bold',
+          textTransform: 'capitalize',
+        }}
+      >
+        {priority} Priority Alert
+      </AlertTitle>
+      <Box sx={{ p: 2 }}>
+        {shouldShowClientName && (
+          <Typography variant='body2' sx={{ pb: 1 }}>
+            {clientNameAllParts(client)}
+          </Typography>
+        )}
+        <Typography variant='body1' sx={{ pb: 1 }}>
+          {alert.note}
+        </Typography>
+        <Typography variant='body2' sx={{ fontSize: '12px' }}>
+          Created by {alert.createdBy?.name} on{' '}
+          {parseAndFormatDateTime(alert.createdAt)}.
+          {alert.expirationDate &&
+            ` Expires on ${parseAndFormatDateTime(alert.expirationDate)}.`}
+        </Typography>
+      </Box>
+    </Alert>
+  );
+};
+
+export enum AlertContext {
+  Client = 'Client',
+  Household = 'Household',
+}
+
+interface ClientAlertsProps {
+  clients: ClientFieldsFragment[] | HouseholdClientFieldsClientFragment[];
+  alertContext: AlertContext;
+  loading?: boolean;
+}
+
+const ClientAlerts: React.FC<ClientAlertsProps> = ({
+  clients,
+  alertContext = AlertContext.Client,
+  loading = true,
+}) => {
+  const clientAlerts: ClientAlertCardProps[] = [];
+  if (!clients) return;
+  clients.forEach((c) => {
+    if (c.access.canViewClientAlerts) {
+      c.alerts.forEach((a) => {
+        clientAlerts.push({
+          alert: a,
+          client: c,
+          shouldShowClientName: alertContext === AlertContext.Household,
+        });
+      });
+    }
+  });
+
+  const title = `${alertContext} Alerts (${clientAlerts.length})`;
+  if (loading && clients.length === 0) return <Loading />;
+
+  return (
+    <TitleCard
+      title={title}
+      headerVariant='border'
+      headerTypographyVariant='body1'
+    >
+      <Box sx={{ m: 2 }}>
+        {clientAlerts.length === 0 &&
+          `${alertContext} has no alerts at this time`}
+        <Stack gap={2}>
+          {clientAlerts.map((ca) => (
```

Comment:
Another related question - I'm assuming we should not show expired alerts at all, and I'm guessing expired alerts should be filtered out already by the time we get here -- e.g. the backend query should just not return expired alerts. Is that right? Should I modify the backend query to not return expired alerts ever, or just not by default but there's some param that allows it if specifically requested?

Reply:
I updated https://github.com/greenriver/hmis-warehouse/pull/3946 to prevent returning expired alerts in the graphql query